clusters/app.ci/prow-monitoring/mixins/_prometheus/release_controller_alerts.libsonnet

```diff
@@ -33,6 +33,38 @@
           }
         ],
       },
+      {
+        name: 'release-controller-git-cache-warning',
+        rules: [
+          {
+            alert: 'releaseControllerGitCacheWarning',
+            expr: '100 * kubelet_volume_stats_available_bytes{job="kubelet",namespace="ci-release",persistentvolumeclaim="git-git-cache-0"} / kubelet_volume_stats_capacity_bytes{job="kubelet",namespace="ci-release",persistentvolumeclaim="git-git-cache-0"} < 5',
```

Comment:
`kubelet_volume_stats_available_bytes` is not in the prometheus instance of prow-monitoring.
It is in the one from `openshift-monitoring`.
However, we cannot define alerts there because `app.ci` is an OSD cluster and the alerts go to the OSD team.

Reply:
Well, that stinks!  Even with cluster-admin rights, It doesnt look like we can create receivers in the OSD clusters. :'(

Reply:
Looks like 4.11 should be able to handle it:
https://issues.redhat.com/browse/OBSDA-3

Reply:
I have something blocked by the feature as well: I would remove all this prow-monitoring deployment of Prometheus/AlertManager and use them in the openshift-monitoring ns. But it is still a long shot.

Even if user defined alerts are implemented, we will have to set up the alerts as defined by some new doc.
This PR changes the things for prow-monitoring.

Reply:
I will open a card on our board to re-add these alerts once their story is complete and we have a better picture of our integration with this new work! 
Thanks @hongkailiu and @petr-muller!

Reply:
Ticket created: https://issues.redhat.com/browse/OCPCRT-131